tests/unit/test_charm_2.py

```diff
@@ -435,3 +455,50 @@ def test_get_initial_password_action(self):
             event.set_results.mock_calls,
             [unittest.mock.call({"password": consensus["default_admin_password"]})]
         )
+
+    def test_theme_reconciliation(self):
+        """
+        arrange: after peer relation established and database ready
+        act: update themes configuration
+        assert: themes installed in WordPress should update according to the themes config
+        """
+        self._setup_replica_consensus()
+        self.harness.update_config({
+            "db_host": "config_db_host",
+            "db_name": "config_db_name",
+            "db_user": "config_db_user",
+            "db_password": "config_db_password",
+        })
+
+        self.assertEqual(
+            self.installed_themes,
+            set(self.harness.charm._WORDPRESS_DEFAULT_THEMES),
+            "installed themes should match the default installed themes "
+            "with the default themes config"
+        )
+
+        # Currently, Ops test framework has a bug preventing starting an already
```

Comment:
Excellent documentation for this! Do you think a date might help as well just to trigger a possible review if it has been a while?

Reply:
I will take a look at this problem this week to confirm if this problem exists and maybe submit a issue or PR for this.